var/spack/repos/builtin/packages/py-submitit/package.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PySubmitit(PythonPackage):
+    """Python toolbox for submitting jobs to Slurm."""
+
+    homepage = "https://github.com/facebookincubator/submitit"
+    pypi = "submitit/submitit-1.3.3.tar.gz"
+
+    version("1.3.3", sha256="efaa77b2df9ea9ee02545478cbfc377853ddf8016bff59df6988bebcf51ffa7e")
+
+    depends_on("python@3.6:", type=("build", "run"))
+    depends_on("py-setuptools", type=("build", "run"))
```

Comment:
Are you sure it's needed at run-time?

Reply:
I can confirm that `setuptools` is needed at run-time because `pkg_resources` is imported [here](https://github.com/facebookincubator/submitit/blob/6f9e1f67178b08b050576fe6bc02e4555568128a/submitit/core/plugins.py#L24).